pkg/util/validate/additional_scopes.go

```diff
@@ -0,0 +1,16 @@
+package validate
+
+import (
+	"github.com/greenpau/go-authcrunch/pkg/errors"
+	"regexp"
+)
+
+// AdditionalScopes verifies if the provided additional_scopes argument is valid
+func AdditionalScopes(additionalScopes string) error {
+	compile, _ := regexp.Compile("^[\\w|\\s]+$")
+	match := compile.MatchString(additionalScopes)
```

Comment:
@dbranco-sdc , could just be `if compile.MatchString ...`. Please rename `compile` to `pattern` or `r`.